content/en/docs/tutorials/pod-security/cluster-level-pss.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Applying Pod Security Standards at Cluster level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Disclaimer" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
```

Comment:
Start with an overview. This is a one-three sentences that explain what the tutorial teaches you to do.  Something like "This tutorial shows you how to use Pod Security Admission to apply Pod Security Standards to new Kubernetes clusters." (this would remove the need for the Note at the start too)